nav2_bt_navigator/src/navigators/navigate_to_pose.cpp

```diff
@@ -101,7 +101,8 @@ NavigateToPoseNavigator::goalReceived(ActionT::Goal::ConstSharedPtr goal)
 }
 
 void
-NavigateToPoseNavigator::goalCompleted(typename ActionT::Result::SharedPtr /*result*/)
+NavigateToPoseNavigator::goalCompleted(typename ActionT::Result::SharedPtr /*result*/,
+                                       nav2_behavior_tree::BtStatus /*final_bt_status*/)
```

Comment:
This is not linted properly, there are no line alignments in C++ (that's python), this should just be a new line with 2 space indent

Reply:
Also, make BTStatus `const`